docs/product-model.md

```diff
@@ -2,6 +2,16 @@
 title: Product Model
 ---
 
+---
+**Note:**
```

Comment:
These comments should be temporary and removed once work is complete. The documentation is meant to describe the CDM "as it is" (in that released version), and not refer to any past, future or on-going development dynamics.

Reply:
Due to the phased delivery of the enhancements, the interim version of the documentation needs some comments.  These will be removed from the final PR before the merge to production,